commands/task_done.py

```diff
@@ -0,0 +1,57 @@
+
+from models import *
+from helpers.errorhelper import ErrorHelper
+import select
+
+class TaskDone:
+    def __init__(self, data):
+        self.data = data
+        self.payload = {
+            "response_type": "ephemeral",
+            "blocks": []
+        }
+    
+    #if get user id of given slack id/ create if not exists and return user id
+    def get_or_create(self, uid):
```

Comment:
Ideally this function should be in User class
so others can also reuse this function if they want to user get_or_create on User model